code/datums/quirks/negative.dm

```diff
@@ -10,17 +10,17 @@
 	lose_text = "<span class='notice'>Your back feels better.</span>"
 	medical_record_text = "Patient scans indicate severe and chronic back pain."
 	hardcore_value = 4
-	mail_goodies = list(/obj/item/cane)
+	mail_goodies = list(/obj/item/storage/briefcase)
```

Comment:
Why?

Reply:
When I did the mail quirk DLC, I had the bad back quirk PR halfway done and the cane was supposed to be a supporting item for it.  When I talked to Arm he wanted bad back to force lawyers to use briefcases so this is the reason for me switching the items.

Reply:
Put this in the changelog

Reply:
Done.